python/py-h11/Portfile

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-h11
+version             0.7.0
+
+categories-append   net www
+platforms           darwin
+supported_archs     noarch
+license             MIT
+
+maintainers         {gmail.com:yan12125 @yan12125} openmaintainer
+
+description         A pure-Python, bring-your-own-I/O implementation of HTTP/1.1
+long_description    \
+    This is a little HTTP/1.1 library written from scratch in Python, heavily \
+    inspired by hyper-h2.
+
+homepage            https://github.com/njsmith/h11
+
+distname            ${python.rootname}-${version}
+use_zip             yes
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}/
+
+checksums           md5     e411d454aa515cf9cbd43cd12999b3d2 \
+                    rmd160  549b548f195333573f7b3a6f4157af33303b0b27 \
+                    sha256  1c0fbb1cba6f809fe3e6b27f8f6d517ca171f848922708871403636143d530d9
+
+
+python.versions     27 36
+
+if {${name} ne ${subport}} {
+    depends_build-append    port:py${python.version}-setuptools
+
+    depends_test-append     port:py${python.version}-pytest
+
+    test.run                yes
+    test.cmd                py.test-${python.branch}
+    test.target             h11
+
+    livecheck.type  none
+} else {
+    livecheck.type  regex
+    livecheck.url   https://pypi.python.org/pypi/${python.rootname}/json
+    livecheck.regex "\"${python.rootname}-(\[.\\d\]+)\\${extract.suffix}\""
```

Comment:
I wonder what goes wrong with default values. I'm sure there must be something that should make these regexps less painful than this. There's `livecheck.type pypi` etc., but I would need to look it up in other ports.

Reply:
Ah, I just copied these lines from another port without thinking again :)

Yes livecheck.type pypi is a better approach. Changed all ports involved in this PR.